.github/workflows/antsibull-docs.yml

```
@@ -74,6 +74,12 @@ jobs:
           cat build.sh
         working-directory: antsibull-docs
 
+      - name: Patch requirements.txt to remove antsibull-docs requirement
+        run: |
+          sed -i requirements.txt -e 's!antsibull-docs[^a-zA-Z0-9_-].*!!g'
```

Nit: Why not delete the line entirely?

```suggestion
          sed -i requirements.txt -e '/antsibull-docs[^a-zA-Z0-9_-].*/d'
```